src/components/dashboard/Dashboard.tsx

```diff
@@ -1,44 +1,39 @@
 'use client'
 import { useState } from 'react'
-import type { Subscribers, DealsRecord } from '@/xata'
-import DealsList from '../deals/DealsList'
-import SubscriberList from '@/components/subscriber/SubscriberList'
+// import type { Subscribers, DealsRecord } from '@/xata'
+// import DealsList from '../deals/DealsList'
+// import SubscriberList from '@/components/subscriber/SubscriberList'
+import Link from 'next/link'
 
 interface DashboardProps {
-  deals: any
```

Comment:
We should remove the props here. They will be handled by the other components

Reply:
Will be rolled into next push